### ISSUE TITLE
 BingMaps layer fails to load in WFS example

### ISSUE THREAD

**Issue by chrismayer** [openlayers/openlayers#10774]
**Describe the bug**
The `BingMaps` layer in the online [WFS example](https://openlayers.org/en/latest/examples/vector-wfs.html) seems to broken. It seems that the credentials do not work anymore (excerpt of the server response):

```
olc_5 (
{
"authenticationResultCode": "DeniedCredentials",
"brandLogoUri": "http://dev.virtualearth.net/Branding/logo_powered_by.png",
"copyright": "Copyright © 2020 Microsoft and its suppliers. All rights reserved. This API cannot be accessed and the content and any results may not be used, reproduced or transmitted in any manner without express written permission from Microsoft Corporation.",
"errorDetails": [
"The request was forbidden.  Your credentials may be denied or suspended."
],
"resourceSets": [],
"statusCode": 403,
"statusDescription": "Forbidden",
"traceId": "89a1477c6a834b91842d9a9133ae817e|DU00000D74|0.0.0.1"
}
)
```
This might be related to other examples as well, which are using the BingMaps layer.
 
**To Reproduce**
1. Go to https://openlayers.org/en/latest/examples/vector-wfs.html
2. See that the background map does not load and thus is not visible

**Expected behavior**
The  `BingMaps` layer should be visible in the WFS example.

**Comment by mike-000** [comment]
None of the Bing examples works due to the API key being regularly overloaded.  Most have already been replaced by MapTiler imagery, so only the Bing maps and Preload examples should need the API key.  There is already a PR to update the WFS examples.

**Comment by fredj** [comment]
fixed with #10767
